fairytale.md

```diff
@@ -3,10 +3,42 @@
 ------------------------------------
 
 Although many had tried, no one succeded again. 
+<<<<<<< HEAD
+
+=======
```

Comment:
Hi @gallenaxel . There is leftover markers from the merge. They should be removed when making the merge. What exactly did you do? I feel you may not have understood how this works. If you compare with my main (https://github.com/floridop/MNXB01-fairytale/blob/main/fairytale.md) you will notice these are not there.

Those two markers are at the two edges of the content that is exclusive to your `mystory` branch.